### DISCUSSION TITLE
how to blacklist a directory for all profiles?

### DISCUSSION THREAD

**Discussion by danielkrajnik** [netblue30/firejail#4817] [Q&A] [answered]
Is there a way to blacklist a `/secret-dir` folder, so that all apps with profiles will include it?

**Comment by rusty-snake** [comment, marked as answer]
https://github.com/netblue30/firejail/wiki/Creating-Profiles#locations-and-types

> - **`globals.local`** is a special override file. It will affect every regular profile.

**Reply by danielkrajnik** [reply]
perfect, thank you very much for the prompt and accurate response. I thought that it was the case, but running programs with firejail didn't show anything like `Reading profile globals.profile`. Good to know that it is the case

EDIT: `globals.local` not `globals.profile`

**Reply by rusty-snake** [reply]
> didn't show anything like `Reading profile global.profile`

global.profile != globals.local ...
